### PR TITLE
fix:[PIPE-22410]:Added details on GitLab repository search limitations

### PR DESCRIPTION
**Pull request by krishi0408**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/platform/git-experience/oauth-integration.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ In case of [Github](https://docs.github.com/en/apps/oauth-apps/building-oauth-ap
 
 ### Gitlab
 
-For [GitLab](https://docs.gitlab.com/ee/integration/oauth_provider.html#adding-an-application), users need to ensure that their Vanity URL is added to the OAuth application's redirect URI list. They must provide their subdomain to have the Vanity URL whitelisted. Contact [Harness Support](mailto:support@harness.io) to complete this process.
+For [GitLab](https://docs.gitlab.com/ee/integration/oauth_provider.html#adding-an-application), users need to ensure that their Vanity URL is added to the OAuth application's redirect URI list. Customer must provide their subdomain to have the Vanity URL whitelisted. Contact [Harness Support](mailto:support@harness.io) to complete this process.
```
Comment (senjucanon2) [inline docs/platform/git-experience/oauth-integration.md:90]: change it to "Customers"

**Review by senjucanon2 — CHANGES_REQUESTED** [review]

**Review by senjucanon2 — DISMISSED** [review]
LGTM

**Review by mohitg0795 — DISMISSED** [review]

**Review by senjucanon2 — APPROVED** [review]
LGTM

**Review by Debanitrkl — APPROVED** [review]
LGTM

**Pull request merged**